integration-scripts/src/echoServer.ts

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+import express from 'express'
+import bodyParser from 'body-parser'
+import chalk from 'chalk'
+
+let echoes = 0
+const app = express()
+app.use(bodyParser.json())
+
+app.get('/count', function(_, res) {
+  res.json(echoes)
+})
+
+app.all('*', function(req, res) {
+  echoes += 1
+  const { headers, body } = req
+
+  console.log({ headers, body })
```

Comment:
How do you feel about logging these with a prefix so that you can search for common value when debugging? e.g.

```
console.log('Headers: %o', headers)
console.log('Body: %o', body)
```

Reply:
Actually just removed this, dont really need it